test/babel-hook.js

```diff
@@ -8,6 +8,7 @@
 require("babel/register")({
     // Let babel know to transpile JSX into E(...)
     jsxPragma: "E",
+    optional: ["es7.functionBind"],
```

Comment:
It's a bummer that the config here (for use with mocha), and the config in our `webpack.config.js` are two separate things.